modules/emoji/emoji_data.go

```diff
@@ -6,7 +6,6 @@ package emoji
 
 // Code generated by gen.go. DO NOT EDIT.
 // Sourced from https://raw.githubusercontent.com/github/gemoji/master/db/emoji.json
-//
```

Comment:
This means that this file won't match the generation anymore - we would need to change `generate_emoji` too.

Reply:
Done